roman-numerals-solution/RomanNumeralsConvert.java

```diff
@@ -0,0 +1,167 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class RomanNumeralsConvert {
+	
+	public final static String onesArray[] = { "I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX" };
+	public final static String tensArray[] = { "X", "XX", "XXX", "XL", "L", "LX", "LXX", "LXXX", "XC" };
+	public final static String hundredsArray[] = { "C", "CC", "CCC", "CD", "D", "DC", "DCC", "DCCC", "CM" };
+
+	public final static String TEST_MOD = "-test";
+	public final static String ONE = "I";
+	public final static String TWELVE = "XII";
+	public final static String ONE_HUNDRED_AND_THREE = "CIII";
+
+	private static String[] numArray;
```

Comment:
You either solve this problem in an OOP fashion or functional, but this is approach is none of them.